x/feegrant/keeper/msg_server.go

```diff
@@ -84,3 +85,21 @@ func (k msgServer) RevokeAllowance(ctx context.Context, msg *feegrant.MsgRevokeA
 
 	return &feegrant.MsgRevokeAllowanceResponse{}, nil
 }
+
+// PruneAllowances removes expired allowances from the store.
+func (k msgServer) PruneAllowances(ctx context.Context, req *feegrant.MsgPruneAllowances) (*feegrant.MsgPruneAllowancesResponse, error) {
+	err := k.RemoveExpiredAllowances(ctx, 75)
```

Comment:
small comment on 75, even saying it was arbitrary works

Reply:
how much gas would this consume?

Reply:
@tac0turtle , can someone try to prune the grant by paying gas? I'm not sure if this message could be used much. or should we consider pruning the grants of msg.sender when this message is got executed?

Reply:
I don't think it would make much sense to prune a specific user's grants, because it serves no purpose to them. This message is like doing work for the chain

Reply:
Gas consumed per prune

50: 112450
75: 168675
100: 224900
150: 337350

Reply:
> @tac0turtle , can someone try to prune the grants (specifically someone else's grants) by paying gas ? I'm not sure if this message could be used much. or should we consider pruning the grants of msg.sender when this message gets executed?

this is meant to prune only expired grants, there should not be a way for users to prune existing grants

Reply:
> Gas consumed per prune
> 
> 50: 112450 75: 168675 100: 224900 150: 337350

looks good, lets leave a comment saying 75